CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 *                                           @box/webapp
 
 # Root
-/*                                          @box/foundation
+/*                                          @box/ui-elements
```

Comment:
Removed `foundation` and pass ownership to `ui-elements` to all security bumps and  other enhancements since we have more bandwidth

Reply:
Do we want to keep `foundation` and just include `ui-elements`?

```suggestion
/*                                          @box/foundation @box/ui-elements
```

It should act as an OR: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file

Reply:
![Screen Shot 2022-05-11 at 5 41 43 PM](https://user-images.githubusercontent.com/103291617/167952685-e72a8326-38b5-4f76-ab1c-15440b59e098.png)
When this setting is enabled doesnt that turn into an `AND`